testing/src/test/java/io/stargate/it/BaseStorageIntegrationTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.it;
+
+import com.datastax.oss.driver.api.core.Version;
+import com.datastax.oss.driver.api.testinfra.ccm.CcmBridge;
+
+/**
+ * Base class for tests that need a backend database cluster managed by {@code ccm}.
+ *
+ * <p>Note: the cluster is created on class initialization and is destroyed at JVM exit.
+ */
+public class BaseStorageIntegrationTest {
+
+  public static final String CLUSTER_NAME = "Test_Stargate_Cluster";
+
+  private static final String CCM_VERSION = "ccm.version";
+  private static final boolean EXTERNAL_BACKEND =
+      Boolean.getBoolean("stargate.test.backend.use.external");
+
+  private static final int clusterNodes = Integer.getInteger("stargate.test.backend.nodes", 1);
+  private static final CcmBridge ccm;
+
+  static {
+    String version = System.getProperty(CCM_VERSION, "3.11.8");
+    System.setProperty(CCM_VERSION, version);
+
+    ccm =
+        CcmBridge.builder()
+            .withCassandraConfiguration("cluster_name", CLUSTER_NAME)
+            .withNodes(clusterNodes)
+            .build();
+
+    if (!EXTERNAL_BACKEND) {
+      ccm.create();
+      ccm.start();
+
+      Runtime.getRuntime()
```

Comment:
In driver we have:
```
  Runtime.getRuntime()
        .addShutdownHook(
            new Thread(
                () -> {
                  try {
                    ccmBridge.remove();
                  } catch (Exception e) {
                    // silently remove as may have already been removed.
                  }
                }));
```
so maybe add this `try catch` block

Reply:
sure. will do.

Reply:
Updated and force pushed